### PR TITLE
add vukky autocat

### PR DESCRIPTION
**Pull request by Vukky123**
this is a very nice PR that should work as far as I know

### CODE REVIEW

File: eod/categories.go [hunk 1]
```diff
@@ -15,10 +15,11 @@ var autocats = map[string]func(string) bool{
 	"Cheesy":     func(s string) bool { return strings.HasPrefix(strings.ToLower(s), "cheesy") },
 	"Bloops":     func(s string) bool { return strings.HasSuffix(strings.ToLower(s), "bloop") },
 	"Melons":     func(s string) bool { return strings.HasSuffix(strings.ToLower(s), "melon") },
-	"Numbers": func(s string) bool {
+	"Numbers":    func(s string) bool {
 		_, err := strconv.ParseFloat(s, 32)
 		return err == nil
 	},
+	"Vukkies":    func(s string) bool { return strings.HasPrefix(strings.ToLower(s), "vukky") || strings.HasSuffix(strings.ToLower(s), "vukky") },
```
Comment (Nv7-GitHub) [inline eod/categories.go:22]: Can't it just use `strings.Contains` instead?

Comment (Vukky123) [reply eod/categories.go:22]: this is very true, thank you for reminding me

File: eod/categories.go [hunk 2]
```diff
@@ -15,10 +15,12 @@ var autocats = map[string]func(string) bool{
 	"Cheesy":     func(s string) bool { return strings.HasPrefix(strings.ToLower(s), "cheesy") },
 	"Bloops":     func(s string) bool { return strings.HasSuffix(strings.ToLower(s), "bloop") },
 	"Melons":     func(s string) bool { return strings.HasSuffix(strings.ToLower(s), "melon") },
-	"Numbers": func(s string) bool {
+	"Numbers":    func(s string) bool {
 		_, err := strconv.ParseFloat(s, 32)
 		return err == nil
 	},
+	"Vukkies":    func(s string) bool { return strings.Contains(strings.ToLower(s), "vukky") },
+	"Fukkies":    func(s string) bool { return strings.Contains(strings.ToLower(s), "fukky") },
```
Comment (Nv7-GitHub) [inline eod/categories.go:23]: No swear words pls

**Review by Nv7-GitHub — CHANGES_REQUESTED** [review]

File: eod/categories.go (hunk 2 above)
Comment (Vukky123) [reply eod/categories.go:23]: alrighty, the fukky category was more of a joke @vtheskeleton did anyway

**Review by Nv7-GitHub — APPROVED** [review]

**Comment by Nv7-GitHub** [comment]
Looks ok!

File: eod/categories.go (hunk 2 above)
Comment (vtheskeleton) [reply eod/categories.go]: > alrighty, the fukky category was more of a joke @vtheskeleton did anyway

you dare drag me into this

**Pull request merged**